From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Subject: [PATCH] Unlock Rust toolchain

This is meant to match Deno's rust-toolchain.toml, so it's useless to us.

---

diff --git a/rust-toolchain.toml b/rust-toolchain.toml
deleted file mode 100644
index 0924897..0000000
--- a/rust-toolchain.toml
+++ /dev/null
@@ -1,12 +0,0 @@
-[toolchain]
-channel = "1.91.0"
-components = ["rustfmt", "clippy"]
-targets = [
-    "x86_64-apple-darwin",
-    "aarch64-apple-darwin",
-    "x86_64-unknown-linux-gnu",
-    "aarch64-unknown-linux-gnu",
-    "x86_64-pc-windows-msvc",
-    "aarch64-linux-android",
-    "x86_64-linux-android",
-]
